<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_LineJoinOptions</class>
 <widget class="QWidget" name="LC_LineJoinOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>24</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="9">
    <widget class="QCheckBox" name="cbPolyline">
     <property name="toolTip">
      <string>If checked, polyline will be created instead of individual lines</string>
     </property>
     <property name="text">
      <string>Create Polyline</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QComboBox" name="cbLine2EdgeMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Policy for the line was selected second</string>
     </property>
     <item>
      <property name="text">
       <string>Extend/Trim</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Add segment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No change</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Line 1:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Line 2:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="cbLine1EdgeMode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>95</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Policy for joining line that was selected first</string>
     </property>
     <item>
      <property name="text">
       <string>Extend/Trim</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Add segment</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No change</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QComboBox" name="cbAttributesSource">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Defines how pen should be applied to created entities</string>
     </property>
     <item>
      <property name="text">
       <string>Active Pen</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Line 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Line 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Both lines</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Attributes:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="11">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="10">
    <widget class="QCheckBox" name="cbRemoveOriginals">
     <property name="toolTip">
      <string>If checked, original lines will be removed</string>
     </property>
     <property name="text">
      <string>Remove originals</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
